### PR TITLE
feat: add konflux jobs metadata endpoint

### PR DESCRIPTION
**Pull request by tnevrlka**
Jira: https://issues.redhat.com/browse/KONFLUX-3458

In order to not lose metrics during migration to konflux from openshift ci we need to create an endpoint in Quality Dashboard to receive metrics from Konflux.

### CODE REVIEW

**Review by jsmid1 — APPROVED** [review]

File: backend/api/server/router/konflux/types.go [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package konflux
+
+import "time"
+
+type KonfluxMetadata struct {
```
[the remark this answers is not in the document]
Comment (tnevrlka) [reply backend/api/server/router/konflux/types.go:8]: Thanks for the review, resolved.

**Review by flacatus — APPROVED** [review]

**Pull request merged**